src/prognostic_equations/hyperdiffusion.jl

```diff
@@ -276,31 +277,30 @@ NVTX.@annotate function apply_tracer_hyperdiffusion_tendency!(Yₜ, Y, p, t)
     (; hyperdiff, turbconv_model) = p.atmos
     isnothing(hyperdiff) && return nothing
 
+    # When α_hyperdiff_tracer is 0, precipitating species are not hyperdiffused.
```

Comment:
```suggestion
    # α_hyperdiff_tracer rescales the hyperdiffusivity for precipitating species 
```
(Move this and the following line to above l. 286, where this is being used.)